### ISSUE TITLE
Potential Revival of this Codebase

### ISSUE THREAD

**Issue by Jacob-Steele** [homeaway/jenkins-spock#147]
Is it at all possible to revive this codebase so a new release can be done or ownership can be passed on to people who will continue to maintain it?

A new release would be great to fix the slowness of tests that fixes https://github.com/ExpediaGroup/jenkins-spock/pull/109 implemented

I'd like to look into potentially adding support for Java 17, as well as fix some issues like https://github.com/ExpediaGroup/jenkins-spock/issues/143 so we can bump the Jenkins versions up in our tests.

Let me know if I can do anything to help kick this process off.

Thanks,

**Comment by deblaci** [comment]
Yes it is would be nice, we are waiting for the next release since #109 fixed, we had to release it to own maven repo, but community development would be better.
We have also a stub class what implements some 
Jenkins pipeline functionality(env vars, params, retry impl.) what can be incorporated to this project.

Or Are there any fork of this what actively developed?

**Comment by Jacob-Steele** [comment]
@deblaci I may wait a bit to see if any response comes in from the original authors.

If nothing comes around, I'll try and see if I can get it forked somewhere to add some more contributions to.

**Comment by balza** [comment]
@Jacob-Steele have you forked the project? If so I like give an hand.

**Comment by Jacob-Steele** [comment]
@balza No I haven't had a chance to fork this repo and attempt to revive it. 

One of the reasons I wanted to attempt to revive this was due to my organization using this project to do heavy lifting in tests related to Jenkins. We were able to commit time to looking at forking and improving this repo. Unfortunately, priorities changed for my org to shift away from focusing on Jenkins to other CI/CD platforms and caused this to fall off our radar.

If anything does come of being forked on this project, I can help contribute towards it, I just don't have the personal time right now to be a key contributor in it.

**Comment by slmjy** [comment]
My org will also be interested in contributing

**Comment by balza** [comment]
@awittha do you have any suggestion??

**Comment by SaranshIam** [comment]
+1 We would like to contribute in maintaining this.

**Comment by Gengar003** [comment]
Hi, I am @awittha, however, I have left ExpediaGroup and no longer have access to the @awittha account so I can't prove that (nor manage this repository any longer). I also no longer have access to, nor use, Jenkins, so my ability to contribute meaningfully to the codebase is gone for now.

I would recommend forking the repository and all collaborating in the new one, in something not tied to a corporate employee's account... unless some other ExpediaGroup employee happens along to handle the revitalization of the codebase in *this* repository! _(IMO, unlikely....)_

**Comment by albertocavalcante** [comment]
Anyone had any luck on this? Is there an active fork? 

Edit: just sent an email to oss@expediagroup.com asking about the possibility to donate or open to community maintenance.

**Comment by slmjy** [comment]
> Anyone had any luck on this? Is there an active fork?
> 
> Edit: just sent an email to [oss@expediagroup.com](mailto:oss@expediagroup.com) asking about the possibility to donate or open to community maintenance.

We are migrating to [JenkinsPipelineUnit with JUnit](https://github.com/jenkinsci/JenkinsPipelineUnit)
